Basic-Car-Maintenance/Shared/Settings/Views/ContributorsListView.swift

```diff
@@ -8,18 +8,17 @@
 import SwiftUI
 
 struct ContributorsListView: View {
-    
     var viewModel: SettingsViewModel
-    
+
     var body: some View {
         List {
-            if let contributors = viewModel.contributors, !contributors.isEmpty {
-                ForEach(contributors) { contributor in
+            if !viewModel.sortedContributors.isEmpty {
+                ForEach(viewModel.sortedContributors, id: \.id) { contributor in
                     Link(
                         destination: URL(string: contributor.htmlURL) ??
-                        viewModel.urls["Basic-Car-Maintenance"]!) {
-                            ContributorsProfileView(name: contributor.login, url: contributor.avatarURL)
-                        }
+                            viewModel.urls["Basic-Car-Maintenance"]!) {
```

Comment:
All the indentation is off
You can do `Cmd + A` to select everything and `Ctrl + i` to indent everything properly